From 865c90806b2748f97d006279f99cadfa20009543 Mon Sep 17 00:00:00 2001
From: Daniele Palmas <dnlplm@gmail.com>
Date: Thu, 18 Nov 2021 12:09:46 +0100
Subject: [PATCH 09/12] bus: mhi: pci_generic: set wake get/put as no-op for
 FN980

FN980 does not properly support wake_db register related
operations, so set wake get/put as no-op.

Signed-off-by: Daniele Palmas <dnlplm@gmail.com>
---
 drivers/bus/mhi/pci_generic.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/bus/mhi/pci_generic.c b/drivers/bus/mhi/pci_generic.c
index 1363edd0e26b..18646c20eef4 100644
--- a/drivers/bus/mhi/pci_generic.c
+++ b/drivers/bus/mhi/pci_generic.c
@@ -476,7 +476,8 @@ static const struct mhi_pci_dev_info mhi_telit_fn980_info = {
 	.edl = "qcom/sdx55m/edl.mbn",
 	.config = &modem_telit_fn980_config,
 	.bar_num = MHI_PCI_DEFAULT_BAR_NUM,
-	.dma_data_width = 32
+	.dma_data_width = 32,
+	.sideband_wake = true,
 };
 
 static const struct mhi_channel_config mhi_telit_fn980_hw_v1_channels[] = {
-- 
2.32.0

